crates/storage/codecs/derive/src/compact/mod.rs

```diff
@@ -162,7 +162,7 @@ pub fn get_bit_size(ftype: &str) -> u8 {
     match ftype {
         "bool" | "Option" => 1,
         "TxType" => 2,
-        "u64" | "BlockNumber" | "TxNumber" | "ChainId" => 4,
+        "u64" | "BlockNumber" | "TxNumber" | "ChainId" | "TransitionId" | "NumTransactions" => 4,
```

Comment:
Was this a bug?

Reply:
Not exactly, it is needed so that macro can recognize the types.